### DISCUSSION TITLE
What version is skoviox patched on?

### DISCUSSION THREAD

**Discussion by ghost** [3kh0/ext-remover#1065] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
SKOVIOX ---i am on v120 but i legit tried ALL techniques for like 1 hour and they did  not work. eg. when i went to diagnostics and went to settings - it did NOTHING. eg2. when i used resources on ChromeVox - did nothing. can u guys help??? any ideas???
i only have 2 kiosk apps: 1 works, but the other auto-connects to network when opened. any other ways to browse freely? let me know!!!

**Comment by ohiobombs** [comment]
118 works for me and i think i the latest

**Comment by ghost** [comment, marked as answer]
nope. latest is 120.

**Reply by ghost** [reply]
3kh0 HELP!!

**Reply by AshtonDavies** [reply]
There are later versions, but they are more limited on support.

**Reply by ghost** [reply]
my board kinda sucks so they put like idk 1000 restrictions on our devices. the closest I have got to bypassing - a proxy.

**Reply by ghost** [reply]
ps  120 is my version

**Reply by Blobby-Boi** [reply]
do the 124 method on 120

**Reply by Blobby-Boi** [reply]
It should work as far as I know

**Reply by ghost** [reply]
what is the 124 method?

**Reply by Blobby-Boi** [reply]
![image](https://github.com/3kh0/ext-remover/assets/131911782/c8ebf38e-cb50-4e82-889f-c2ae2ebbf1c6)

**Reply by Blobby-Boi** [reply]
sry for bad quality discord screenshot I just dont feel like typing that again

**Reply by ghost** [reply]
thanks so  much boi!

**Reply by ghost** [reply]
never mind, it no work. I'm in the Toronto district school board. its massive (and big-brain)

**Reply by ghost** [reply]
btw I'm a big fan blobii

**Reply by Blobby-Boi** [reply]
why didn't it work? What went wrong?

**Reply by ghost** [reply]
it doesnt let me open settings!

**Reply by FunnyCorgi** [reply]
I tried just now(literally two hours ago, just before my math check-in) and the open in settings link has been removed.

**Reply by FunnyCorgi** [reply]
I am on version 121 btw

**Reply by ghost** [reply]
im in v120

**Reply by VFXKLIPS** [reply]
Yo do you know how to get skiovox back on v121 because the alt shift s doesn't work when I literally load the kiosk app.

**Reply by AshtonDavies** [reply]
> Yo do you know how to get skiovox back on v121 because the alt shift s doesn't work when I literally load the kiosk app.

Most v121 methods do not involve Alt+Shift+S. There is a channel for that [in this server](https://discord.com/channels/916775659731300412/1225153173283672295/1225153173283672295), or you could look around in [Titanium Network](https://discord.gg/unblock). I haven't made instructions for it myself.

**Reply by VFXKLIPS** [reply]
Are u sure this will work tho

**Reply by VFXKLIPS** [reply]
Are u sure this will work tho

**Reply by AshtonDavies** [reply]
You could probably find instructions that are a bit more relevant in TN.

**Reply by VFXKLIPS** [reply]
The discord server doesn't work, it says no text channels 
![Screenshot 2024-05-21 3 12 43 PM](https://github.com/3kh0/ext-remover/assets/167250630/ce44822b-b3b3-473a-92ae-96416e532f0c)

**Reply by VFXKLIPS** [reply]
The discord server doesn't work to it says this 
![Uploading Screenshot 2024-05-21 3.14.31 PM.png…]()

**Reply by AshtonDavies** [reply]
> The discord server doesn't work, it says no text channels 
> ![Screenshot 2024-05-21 3 12 43 PM](https://github.com/3kh0/ext-remover/assets/167250630/ce44822b-b3b3-473a-92ae-96416e532f0c)
> 

Here's the join link, then you can use the text channel link: https://discord.com/invite/xZXFb9gXew
